modules/config-fields-redis.adoc

```diff
@@ -59,9 +55,11 @@ BUILDLOGS_REDIS:
     host: quay-server.example.com
     password: strongpassword
     port: 6379
+    ssl: false
```

Comment:
`ssl: false` and `password: somepassword` cannot exist together. You can have:
```
ssl: false
```
or 
```
ssl: true
password: somepassword
```